mne/_fiff/open.py

```diff
@@ -309,7 +315,7 @@ def _show_tree(
         for k, kn, size, pos, type_ in zip(kinds[:-1], kinds[1:], sizes, poss, types):
             if not tag_found and k != tag_id:
                 continue
-            tag = Tag(k, size, 0, pos)
+            tag = Tag(k, size, 0, -1, pos)
```

Comment:
is this order correct?  `Tag` params order is `kind`, `type`, `size`, `next`, `pos`, looks like `type` and `size` are reversed here